runtime/peaq-dev/src/lib.rs

```diff
@@ -463,66 +467,6 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = ();
 }
 
-parameter_types! {
```

Comment:
I think we should keep the Council and Treasury pallet.

Reply:
Of course! Don't know how that happend... that's a mistake!